src/packages/src/xroad/default-configuration/devices.ini

```diff
@@ -43,6 +43,12 @@
 ;   Specifies private key allowed mechanisms. Supported values:
 ;   CKM_RSA_PKCS, CKM_SHA256_RSA_PKCS, CKM_SHA384_RSA_PKCS, CKM_SHA512_RSA_PKCS, and
 ;   CKM_RSA_PKCS_PSS, CKM_SHA256_RSA_PKCS_PSS, CKM_SHA384_RSA_PKCS_PSS, CKM_SHA512_RSA_PKCS_PSS.
+; slot_ids = STRING LIST (optional, string separator is ",")
```

Comment:
The slot id's are defined to be unsigned long integers by the PKCS11 spec., see http://docs.oasis-open.org/pkcs11/pkcs11-base/v2.40/os/pkcs11-base-v2.40-os.html#_Toc72656028